wiki/Ranking/id.md

```diff
@@ -0,0 +1,41 @@
+---
+stub: true
+tags:
+  - pp
+  - leaderboards
+  - ranking
+  - peringkat
+  - papan peringkat
+---
+
+# Peringkat
+
+**Peringkat** (atau "papan peringkat") adalah sebuah daftar yang mengurutkan anggota komunitas osu! berdasarkan kriteria tertentu.
+
+Semua peringkat kecuali peringkat seseorang pada sebuah beatmap dapat dilihat di menu `peringkat` di header situs osu!. Untuk peringkat seseorang pada sebuah beatmap, dapat dilihat di laman info beatmap.
+
+## Peringkat performance point
+
+*Lihat juga: [Performance point](/wiki/Performance_points)*
+
+Peringkat performance point mendeskripsikan tingkat keahlian pemain sebagai perbandingan dengan pemain lainnya. Papan peringkat ini ditentukan berdasarkan total dari performance point yang dimiliki oleh seorang pemain, dan dapat dilihat di profil pemain tersebut dan di [papan peringkat performa](https://osu.ppy.sh/rankings/osu/performance).
+
+## Peringkat skor
+
+Peringkat skor digunakan untuk membandingkan total skor pemain dan mengurutkannya. Untuk mendapatkan peringkat yang tinggi pada papan peringkat membutuhkan waktu dan komitmen yang besar, tetapi membutuhkan sedikit keahlian mekanis dibandingkan dengan peringkat performance point.
+
+## Peringkat Kudosu
+
+Peringkat Kudosu digunakan untuk membandingkan [Kudosu](/wiki/Modding/Kudosu) yang telah dikumpulkan oleh pengguna dan mengurutkannya. Peringkat ini tidak membutuhkan keahlian bermain, karena Kudosu diperoleh dari [proses modding](/wiki/Modding).
+
+## Peringkat Spotlights
+
+Peringkat Spotlights adalah papan peringkat dengan waktu terbatas yang membandingkan keahlian para pemain berdasarkan sekumpulan beatmap tertentu pada setiap musim [Spotlights](/wiki/Beatmap_Spotlights). Pemain yang memiliki peringkat yang tinggi pada peringkat Spotlights akan menerima penghargaan khusus sebagai hasil dari kerja keras mereka.
+
+## Peringkat beatmap
+
+Peringkat beatmap digunakan untuk membandingkan skor para pemain pada suatu beatmap. Peringkat tersebut akan muncul setelah beatmap [Qualified](/wiki/Beatmaps#qualified), [Ranked](/wiki/Beatmaps#ranked), [Approved](/wiki/Beatmaps#approved), atau Loved. Posisi delapan teratas ditampilkan tanpa harus menggulir kebawah, dan jika pemain memiliki skor dengan posisi 50 teratas, secara otomatis akan masuk ke daftar peringkat. Peringkat dengan posisi pertama dapat dilihat di halaman profil pengguna.
```

Comment:
the links have changed. @SeverinoAji is this translation otherwise ready?
```suggestion
Peringkat beatmap digunakan untuk membandingkan skor para pemain pada suatu beatmap. Peringkat tersebut akan muncul setelah beatmap [Qualified](/wiki/Beatmap/Category#qualified), [Ranked](/wiki/Beatmap/Category#ranked), [Approved](/wiki/Beatmap/Category#approved), atau Loved. Posisi delapan teratas ditampilkan tanpa harus menggulir kebawah, dan jika pemain memiliki skor dengan posisi 50 teratas, secara otomatis akan masuk ke daftar peringkat. Peringkat dengan posisi pertama dapat dilihat di halaman profil pengguna.
```